UnitsNet/CustomCode/Quantities/VolumeFlow.extra.cs

```diff
@@ -36,7 +36,7 @@ public partial struct VolumeFlow
 #if !WINDOWS_UWP
         public static Volume operator *(VolumeFlow volumeFlow, TimeSpan timeSpan)
         {
-            return Volume.FromCubicMeters(volumeFlow.CubicMetersPerSecond * timeSpan.Seconds);
+            return Volume.FromCubicMeters(volumeFlow.CubicMetersPerSecond * timeSpan.TotalSeconds);
```

Comment:
This is good, however it inspired me to do a full search of ".Seconds" in our code base and it turns out we have MANY of these bugs. Could you please address those too?

```
Find all ".Seconds", Match case, Whole word, Subfolders, Find Results 1, Current Project: UnitsNet\UnitsNet.NetStandard10.csproj, "*.cs"
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Angle.extra.cs(47):            return RotationalSpeed.FromRadiansPerSecond(angle.Radians / duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(39):            return time.AddSeconds(duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(44):            return time.AddSeconds(-duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(59):            return duration.Seconds < timeSpan.TotalSeconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(64):            return duration.Seconds > timeSpan.TotalSeconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(69):            return duration.Seconds <= timeSpan.TotalSeconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(74):            return duration.Seconds >= timeSpan.TotalSeconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(79):            return duration.Seconds == timeSpan.TotalSeconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(84):            return duration.Seconds != timeSpan.TotalSeconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(89):            return timeSpan.TotalSeconds < duration.Seconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(94):            return timeSpan.TotalSeconds > duration.Seconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(99):            return timeSpan.TotalSeconds <= duration.Seconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(104):            return timeSpan.TotalSeconds >= duration.Seconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(109):            return timeSpan.TotalSeconds == duration.Seconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(114):            return timeSpan.TotalSeconds != duration.Seconds;
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Duration.extra.cs(119):            return Volume.FromCubicMeters(volumeFlow.CubicMetersPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\KinematicViscosity.extra.cs(57):            return Area.FromSquareMeters(kinematicViscosity.SquareMetersPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\KinematicViscosity.extra.cs(62):            return Area.FromSquareMeters(kinematicViscosity.SquareMetersPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Length.extra.cs(77):            return Speed.FromMetersPerSecond(length.Meters/duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Mass.extra.cs(86):            return MassFlow.FromKilogramsPerSecond(mass.Kilograms/duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\MassFlow.extra.cs(51):            return Mass.FromKilograms(massFlow.KilogramsPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\MassFlow.extra.cs(56):            return Mass.FromKilograms(massFlow.KilogramsPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Power.extra.cs(52):            return Energy.FromJoules(power.Watts * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Power.extra.cs(57):            return Energy.FromJoules(power.Watts * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\RotationalSpeed.extra.cs(52):            return Angle.FromRadians(rotationalSpeed.RadiansPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\RotationalSpeed.extra.cs(57):            return Angle.FromRadians(rotationalSpeed.RadiansPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Speed.extra.cs(57):            return Acceleration.FromMeterPerSecondSquared(speed.MetersPerSecond / duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Speed.extra.cs(62):            return Length.FromMeters(speed.MetersPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Speed.extra.cs(67):            return Length.FromMeters(speed.MetersPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Volume.extra.cs(49):            return VolumeFlow.FromCubicMetersPerSecond(volume.CubicMeters / duration.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\Volume.extra.cs(54):            return VolumeFlow.FromCubicMetersPerSecond(volume.CubicMeters / timeSpan.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\VolumeFlow.extra.cs(39):            return Volume.FromCubicMeters(volumeFlow.CubicMetersPerSecond * timeSpan.Seconds);
  C:\dev\unitsnet\UnitsNet\CustomCode\Quantities\VolumeFlow.extra.cs(44):            return Volume.FromCubicMeters(volumeFlow.CubicMetersPerSecond * duration.Seconds);
  C:\dev\unitsnet\UnitsNet\GeneratedCode\Quantities\Duration.g.cs(902):                }, (x, y) => FromSeconds(x.Seconds + y.Seconds));
  Matching lines: 34    Matching files: 12    Total files searched: 277
```

Reply:
@angularsen Yes, already did. So far we're good with the other overloads :)

Reply:
Hmm.. am I missing something? The search result I pasted above has about 30 more similar issues that this PR does not address, where `.Seconds` is accessed instead of `.TotalSeconds`. This causes problems for timespan values greater than 60 seconds.

Reply:
I'll merge this and fix the others myself as I finally have a some computer time for myself this evening! :-)

Reply:
I think you're seeing UnitsNet.Duration.Seconds which is fine (it is equivalent to TimeSpan.TotalSeconds).

Reply:
I got 34 hits on Duration.Seconds
0 hits on TimeSpan.Seconds

Reply:
You are absolutely right :-) Just figured that out before reading this.